src/locales/be-BY/messages.po

```diff
@@ -3050,10 +3050,6 @@ msgstr "сінхр."
 msgid "syncing"
 msgstr "ідзе сінхр."
 
-#: src/components/plot/add/PlotAddChoosePlotter.tsx:89
-msgid "thirdparty.plotter.disclaimer"
-msgstr ""
-
```

Comment:
This string was inadvertently added when `chiaplotters_gui` was merged. I did a batch find/replace to remove them and then re-ran `npm run locale:extract && npm run locale:compile`